recipe/meta.yaml

```diff
@@ -0,0 +1,149 @@
+    - pytest -n auto -k "not ({{ tests_to_skip }} $DISABLE_LICENSED or test_read_write_splayed or test_read_write_serialized_async or test_read_write_serialized or test_write_json or test_write_splayed or test_read_write_splayed_async  or test_read_write_csv_async or test_read_write_json_async or test_write_serialized or test_read_write_csv or test_read_write_json or test_write_csv)" tests/test_write.py
+    - pytest -n auto -k "not ({{ tests_to_skip }} $DISABLE_LICENSED or test_read_csv or test_read_csv_with_type_guessing or test_read_json or test_read_fixed_async or test_read_serialized or test_read_fixed or test_read_splayed )" tests/test_read.py
+    - pytest -n auto -k "not ({{ tests_to_skip }} $DISABLE_LICENSED or test_import_from_other_dir or test_delitem or test_attributes or test_call_with_params or test_call_sync or test_setitem or test_getitem or test_dir or test_setattr)" tests/test_q.py
+    - pytest -n auto -k "not ({{ tests_to_skip }} $DISABLE_LICENSED or test_ipc_messaging_tcp_auth or test_async_deferred_calls or test_raw_await or test_memory_domain or test_ipc_messaging_tcp_compression_edge_case or test_no_pyfunc_over_ipc or test_raw_complex or test_tls_and_unix_error or test_ipc_messaging_unix or test_async_q_connection_clears_calls_on_close or test_raw_poll_send_recv_all or test_secure_q_con_callable or test_eventloop_gather_order or test_async_timeout or test_ipc_messaging_unix_auth or test_raw_poll_send_recv_one or test_async_q_connection or test_async_with_q_features or test_async_repr or test_no_pykx_namespace or test_async_helpful_error_for_closed_conn or test_max_error_length or test_no_wrap_over_ipc or test_uninitialized_connection or test_no_ctx_not_used or test_repr or test_raw_poll_send_recv_n or test_sync_helpful_error_for_closed_conn or test_ipc_messaging_tcp or test_timeout)" tests/test_ipc.py
+    - pytest -n auto -k "not ({{ tests_to_skip }} $DISABLE_LICENSED or test_QHOME)" tests/test_config.py
+{% endif %}
+    # these are all other tests
+    - pytest -n auto -k "not ({{ tests_to_skip }} $DISABLE_LICENSED)" test/test_cast.py test/test_pandas_apply.py test/test_register.py test/test_sql.py test/test_cloud_edition.py test/test_license.py
+    - pytest -n auto -k "not ({{ tests_to_skip }} $DISABLE_LICENSED)" test/test_pandas_set_index.py  test/test_q_foreign.py  test/test_random.py test/test_exceptions.py  test/test_pandas_api.py  test/test_pykx.py test/test_schema.py
+{% endif %}
+  requires:
+    - pip
+    - plotly
+    - pytest
+    - pytest-asyncio
+    - pytest-benchmark
+    - pytest-xdist
+    - pytest-randomly
+    - psutil
+    - pytest-timeout
+    - pyarrow         # [not s390x]
+{% if install_kdb_as_test_dep %}
+    # kdb is only present on the anaconda.org/kx channel, hence the channel needs to be enabled in abs.yaml
+    # in the future we may agree with kx to distribute this package ourselves.
+    - kdb
+{% endif %}
+
+about:
+  home: https://code.kx.com/pykx
+  license: Apache-2.0
```

Comment:
now talking with @skupr-anaconda  it should be: 
`LicenseRef-Proprietary AND Apache-2.0`

Reply:
fixed. Will open new PR.